docs/topics/api/collections.rst

```diff
@@ -138,8 +140,9 @@ Delete
 .. _`collection-delete`:
 
 .. note::
-    This API requires :doc:`authentication <auth>` and `Collections:Edit`
-    permission to delete collections other than your own.
+    This API requires :doc:`authentication <auth>`. If you have
+    ``Admin:Curation`` permission you can delete any collection belonging to the
+    ``mozilla`` user.
```

Comment:
Needs updating now after the change to prevent delete for Mozilla collections.

Reply:
done in https://github.com/mozilla/addons-server/pull/8651/commits/ee9d000ccde68b019cd938fe213c4224bd053d65